image/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM openjdk:17-jdk-slim
+
+RUN apt update && \
+    apt install -y unzip wget && \
+    mkdir /grasscutter && cd /grasscutter && \
+    wget -O grasscutter.jar https://github.com/Grasscutters/Grasscutter/releases/download/v1.4.3/grasscutter-1.4.3.jar && \
+    wget https://raw.githubusercontent.com/Grasscutters/Grasscutter/development/keystore.p12 && \
+    wget -P /grasscutter/resources https://git.crepe.moe/grasscutters/Grasscutter_Resources/-/archive/main/Grasscutter_Resources-main.zip && \
+    unzip -d /grasscutter/resources /grasscutter/resources/Grasscutter_Resources-main.zip && rm /grasscutter/resources/Grasscutter_Resources-main.zip && \
+    unzip -d /grasscutter/resources /grasscutter/resources/Grasscutter_Resources-main/Grasscutter_Resources-3.3.zip && rm -rf /grasscutter/resources/Grasscutter_Resources-main && \
```

Comment:
Instead of specifically specifying `3.3`, add a variable for it? (not completely familiar with `Dockerfile`)